template-helpers/hbs-helpers.js

```diff
@@ -363,6 +363,14 @@ function getRemoveString(id, args) {
   return LocaleUtils.formatRemoveString(id);
 }
 
+function checkIfInRemovalPilot(args) {
+  const { session } = args.data.root.req;
+  if (session.kanary && session.kanary.onRemovalPilotList) {
```

Comment:
No code change needed, but optional chaining _will_ change your life 😆 
`if (session.kanary?.onRemovalPilotList) {...}`

I'll see myself out.

Reply:
Yes! Wrote this code before you enlightened me to this amazing new syntax - will add.